hnn_core/gui/_viz_manager.py

```diff
@@ -343,6 +343,17 @@ def _dynamic_rerender(fig):
     fig.tight_layout()
 
 
+def _avg_dipole_check(dpls):
+    """Check for averaged dipole, else average the trials"""
+    # Check if there is an averaged dipole already
+    avg_dpls = [d for d in dpls if d.nave > 1]
+    if avg_dpls:
+        dpl = avg_dpls[0]
+    else:
+        dpl = average_dipoles(dpls)
```

Comment:
Is there a potential case where `avg_dpls `is empty? 
If that's the case, checking the `average_dipoles` function, I don't think is handling empty lists.

Reply:
Currently there this function isn't being used in a case where the dipole list is empty. But I can add a check in case this function is used elsewhere in the future. Thanks!